RedirectApplication/RedirectApplication/Controllers/ValuesController.cs

```diff
@@ -39,8 +42,11 @@ public string Get(int id)
         }
 
         // POST api/values
-        public void Post([FromBody]string value)
+        public void Post(HttpRequestMessage request)
         {
+            var someText = request.Content.ReadAsStringAsync().Result;
+            var reader = new JsonTextReader(new StringReader(someText));
+            PostJson content = JsonSerializer.CreateDefault().Deserialize<PostJson>(reader);
```

Comment:
What happened then will be written in another tickets. This one is about getting JSON and deserialization